core/lib/basic_types/src/protocol_version.rs

```diff
@@ -174,6 +174,10 @@ impl ProtocolVersionId {
     pub fn is_post_1_5_0(&self) -> bool {
         self >= &ProtocolVersionId::Version23
     }
+
+    pub fn is_post_sync_layer(&self) -> bool {
+        self >= &ProtocolVersionId::Version25
+    }
```

Comment:
it's not needed for now right? I don't mind keeping it if you think it will be 100% needed later